src/main/java/prefs/GlobalConfig.java

```diff
@@ -18,10 +18,10 @@
 import java.util.Optional;
 
 /**
- * Abstractions for the contents of the global config file.
+ * Abstractions for the contents of the global config file. Extends Config.
```

Comment:
Don't think this extra part is needed. :stuck_out_tongue: And it gets outdated if we change the subtype hierarchy.